pkg/operatorsource/configuring.go

```diff
@@ -68,34 +77,36 @@ func (r *configuringReconciler) Reconcile(ctx context.Context, in *marketplace.O
 		WithOwnerLabel(in).
 		CatalogSourceConfig()
 
-	err = r.client.Create(ctx, cscCreate)
+	err = r.clientMarketplace.Create(ctx, cscCreate)
+
 	if err != nil && !k8s_errors.IsAlreadyExists(err) {
 		r.logger.Errorf("Unexpected error while creating CatalogSourceConfig: %s", err.Error())
 		nextPhase = phase.GetNextWithMessage(phase.Configuring, err.Error())
 
 		return
 	}
 
-	if err == nil {
+	if !k8s_errors.IsAlreadyExists(err) {
 		nextPhase = phase.GetNext(phase.Succeeded)
 		r.logger.Info("CatalogSourceConfig object has been created successfully")
 
 		return
 	}
 
 	// If we are here, the given CatalogSourceConfig object already exists.
+	r.logger.Info("CatalogSourceConfig object already exists")
```

Comment:
If we are going to add this to the logger (which itself seems orthogonal to the purpose of the pr), shouldn't it be a `Debug` statement?